File: Rentacar.API/Rentacar.Entities/VehicleType.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Rentacar.Entities
{
    public partial class VehicleType
    {
        public VehicleType()
        {
            Models = new HashSet<Model>();
        }

        public int VehicleTypeId { get; set; }
        public string VehicleTypeName { get; set; }

        public virtual ICollection<Model> Models { get; set; }
    }
}

File: rentacar.flutter/lib/models/vehicleType.dart
class VehicleType {
  int vehicleTypeId = 0;
  String vehicleTypeName = '';

  VehicleType.fromJson(Map<String, dynamic> vehicleTypeMap) {
    vehicleTypeId = vehicleTypeMap['vehicleTypeId'] ?? 0;
    vehicleTypeName = vehicleTypeMap['vehicleTypeName'] ?? '';
  }
}
